spec/requests/graphql/mutations/events/update_event_spec.rb

```diff
@@ -22,12 +23,13 @@ def query(id)
           updateEvent(input:{
               id: #{id}
               name: "Paleo gluten-free aesthetic everyday farm-to-table slow-carb distillery."
-              date: "Sat, 19 Dec 2020 15:38:49.000000000 UTC +00:00"
+              date: "Sat, 19 Dec 2020"
```

Comment:
May have to look into validations here to ensure we receive the proper format from FE

Reply:
Was this throwing errors?  In previous tests, it seemed like it didn't matter exacttttttlllyyy how they sent the date, that we'd still process it the way we wanted.  That said, it wouldn't be a bad idea to match everything up.  Good catch!

Reply:
I think actually the error was from the random birthday, which since has been overridden. We can prob change back and it would still work.

But I do agree we should match how we'll get it from the FE

Reply:
Removed time here.